AAP-27696 Create Using Ansible Plug-ins guide (#1635)
ansible/aap-docs: downstream/modules/devtools/proc-rhdh-firewall-example-discover.adoc

:_mod-docs-content-type: PROCEDURE

[id="rhdh-firewall-example-discover_{context}"]
= Discovering existing Ansible content for RHEL system roles

Red Hat recommends that you use trusted automation content that has been tested and approved by Red Hat or your organization. 

{HubNameStart} is a central repository for discovering, downloading, and managing trusted content collections from Red Hat and its partners.
{PrivateHubNameStart} provides an on-premise solution for managing content collections.

. Click on the Ansible `A` icon in the {RHDH} navigation panel.
. Click *Discover existing collections*.
. Click *Go to Automation Hub*. 
+
--

> ** If {PrivateHubName} has been configured in the Ansible plug-ins, you will be redirected to your *PrivateHubName* instance.

Reviewer: ```suggestion
** If {PrivateHubName} has been configured in the Ansible plug-ins, you are redirected to your *PrivateHubName* instance.
```

** If {PrivateHubName} has not been configured in the Ansible plug-ins installation configuration,

> you will be redirected to the Red Hat Hybrid Console (RHCC) automation hub.

Reviewer: ```suggestion
you are redirected to the Red Hat Hybrid Console (RHCC) automation hub.
```
